backend/tests/src/tests/json-schemas/shared/error/sign-up-error-response.schema.ts

```diff
@@ -0,0 +1,17 @@
+const signUpErrorResponseSchema = {
+  'type': 'object',
+  'required': ['statusCode', 'error', 'message'],
+  'properties': {
+    'statusCode': {
+      'type': 'integer',
+    },
+    'error': {
+      'type': 'string',
+    },
+    'message': {
+      'type': 'string',
+    },
+  },
+};
+
+export { signUpErrorResponseSchema };
```

Comment:
Remove the quotes around property names (everywhere)

Reply:
This not `sign-up-error`. This is an error format which is the same for all endpoints. Rename it to `error-response.schema.ts` and `errorResponseSchema`.

Also, export it from `json-schema/shared/error/error.ts`, `json-schema/shared/shared.ts` and `json-schema/json-schema.ts`

Reply:
And then in the spec file import it from `json-schema/json-schema`